shared/video-sdk/develop/real-time-transcription/project-implementation/ios.mdx

```diff
@@ -246,11 +246,11 @@ Take the following steps to start, stop and query a text-to-speech task in your
 
 1. **Query task status**
 
-    You [query the status](#query-the-status-of-speech-to-text) of a Real-Time Transcription task using a `GET` request containing the `taskId` and `builderToken` to notify the user of any change. To do this, add the following method to the `ViewController` class:
+    You [query the status](#query-the-status-of-real-time transcription) of a real-time transcription task using a `GET` request containing the `taskId` and `builderToken` to notify the user of any change. To do this, add the following method to the `ViewController` class:
```

Comment:
Just a query please :
You [query the status](#query-the-status-of-real-time transcription)  --- is hyperlink manage the space.